### ISSUE TITLE
Unable to set alt text to empty string

### ISSUE THREAD

**Issue by Nicd** [lustre-labs/lustre#48]
Setting the `alt` attribute to an empty string adds the attribute and removes it instantly. This causes constant modifications to the element (on every update), and the inability to actually set `alt=""` (which I think is a valid way to say "this image is not relevant to screen readers"). As a workaround, `aria-hidden="true"` can be used for images.

<img width="483" alt="Screenshot 2024-02-27 at 16 06 46" src="https://github.com/lustre-labs/lustre/assets/273137/57be65da-931b-44b5-8153-9f7ea2f2ad4b">

Constant churn in DOM element:
https://github.com/lustre-labs/lustre/assets/273137/e65f684d-5203-44dc-b73d-7715401f0b62

**Comment by hayleigh-dot-dev** [comment]
Fixed for next release!
